js/header.js

```diff
@@ -33,6 +42,14 @@ body.onresize = function () {
             body.style.overflow = 'hidden';
         }
     }
+    if (window.innerWidth < 432)
```

Comment:
Split this logic to a separate function to avoid repetition

Reply:
done